packages/core/src/setup/templates/workflow-section.md

```
@@ -0,0 +1,17 @@
+## Maina Workflow
+
+Every feature follows this sequence:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)